lib/routes/api.js

```diff
@@ -49,7 +49,11 @@ const routes = application => {
   // TODO: Tests
   // TODO: Return error if no items are found
   app.get('/api/types/:type', (req, res, next) => {
-    api.ofType(req.query, req.params.type).then(formatted => {
+    const type = app.get('content-types').find(item => {
```

Comment:
use `req.content.types` to be consistent with [other routes](https://github.com/punchcard-cms/punchcard/blob/master/lib/routes/content.js#L42) and to make this easier to break out for testing later.